webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -460,6 +478,10 @@ Kommunicate.richMsgEventHandler = {
     },
     handleFormSubmit: function(e) {
         e.preventDefault();
+    },
+    isValidString: function(regex, string) {
+        regex = new RegExp(regex)
```

Comment:
this looks confusing, are we converting RegExp to RegExp (regex) or string to regex, please name variable accordingly

Reply:
@devashishmamgain 
it is string to regex conversion,  variable renamed.